### DISCUSSION TITLE
Does this component still works?

### DISCUSSION THREAD

**Discussion by IIIdefconIII** [PiotrMachowski/Home-Assistant-custom-components-Google-Keep#34] [Q&A]
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Google-Keep/assets/9917659/02fb039d-5172-4898-96ec-6cf69e373b0c)

**Comment by PiotrMachowski** [comment]
It has been broken for quite some time now (#31), #32 might fix it

**Reply by IIIdefconIII** [reply]
Thank you for you quick reply!
